app/Libraries/UKCP.php

```diff
@@ -140,4 +144,38 @@ public static function getPathForToken($tokenID, $account)
     {
         return self::TOKEN_PATH_ROOT.$account->id.'/'.$tokenID.'.json';
     }
+
+    public function getStandStatus(string $airfield): array
+    {
+        try {
+            $cached = Cache::get($this->getStandStatusCacheKey($airfield));
+            if ($cached) {
+                return $cached;
+            }
+
+            $response = $this->client->get(
+                sprintf('%s/api/stand/status?airfield=%s', config('services.ukcp.url'), $airfield),
+                ['timeout' => 5]
+            );
+            $body = json_decode($response->getBody()->getContents(), true);
+
+            return tap(
+                collect($body['stands'])->sortBy('identifier', SORT_NUMERIC)->values()->toArray(),
+                fn (array $stands) => Cache::put(
+                    $this->getStandStatusCacheKey($airfield),
+                    collect($stands)->sortBy('identifier', SORT_NUMERIC)->values()->toArray(),
```

Comment:
Might be being dumb, but is `->sortBy('identifier', SORT_NUMERIC)->values()->toArray()` happening twice here?

Reply:
Nope, that's me being dumb! b4a19491